docs/pages/quickstart.rst

```diff
@@ -144,6 +144,48 @@ a columnar data format, like a `pandas <https://pandas.pydata.org/>`_ DataFrame.
     devices = patients.devices
     devices_df = pd.DataFrame(devices.to_list())
 
+Similarly to fetching information about patients, you can fetch information about projects,
+and metadata related to the patients within projects (and cohorts).
+
+You can find information about a single project:
+
+.. code-block:: python
+
+    from runeq.resources.project import get_project
+
+    project = get_project(project_id="example_id")
+    print(project.to_dict())
+
+To view all the patients in a project, and their related project metrics you can use the
+following example:
+
+.. code-block:: python
+
+    from runeq.resources.project import get_all_project_patients
+
+    project_patients = get_all_project_patients(project_id="example_id")
+
+    for project_patient in project_patients:
+        print(project_patient)
+        for metric in patient.metrics:
+            print(' ', metric)
+
+        print('')
+
+It may be easier to view a single project patient in a dataframe which you can do by:
+
+.. code-block:: python
+
+    from runeq.resources.project import get_all_project_patients
+
+    project_patients = get_all_project_patients(project_id="example_id")
+
+    for patient in project_patients:
+        df = patient.get_patient_metadata_dataframe()
+        break
```

Comment:
This example seems a little awkward since we break at the first patient, and a for loop isn't quite necessary. Can you instead index into project_patients and call get_patient_metadata_dataframe on that patient